src/store/commands/types.js

```diff
@@ -5,4 +5,10 @@ const LOGIN = {
 };
 const LOGOUT = { SUCCESS: 'LOGOUT_SUCCESS' };
 
-export { LOGIN, LOGOUT };
+const GET_CANDIDATE_LIST = {
+  REQUEST: 'GET_CANDIDATE_LIST_REQUEST',
+  SUCCESS: 'GET_CANDIDATE_LIST_SUCCESS',
+  FAILURE: 'GET_CANDIDATE_LIST_FAILURE',
+};
+
```

Comment:
Please delete types.js file and move GET_CANDIDATE_LIST to file in action folder

Reply:
done